components/org.wso2.carbon.identity.organization.management.role.management.service/src/main/java/org/wso2/carbon/identity/organization/management/role/management/service/RoleManager.java

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright (c) 2022, WSO2 Inc. (http://www.wso2.com).
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.organization.management.role.management.service;
+
+import org.wso2.carbon.identity.organization.management.role.management.service.exception.RoleManagementException;
+import org.wso2.carbon.identity.organization.management.role.management.service.models.PatchOperation;
+import org.wso2.carbon.identity.organization.management.role.management.service.models.Role;
+
+import java.util.List;
+
+/**
+ * Interface for Role Management.
+ */
+public interface RoleManager {
+
+    /**
+     * Add a role
+     *
+     * @param organizationId The ID of the organization where we add the role.
```

Comment:
let's not mention `we` in the comments. change all applicable places.
```suggestion
     * @param organizationId The ID of the organization where the role should be created.
```